apps/src/codegen.js

```diff
@@ -242,8 +242,6 @@ var createCustomMarshalObject = function (interpreter, nativeObj, nativeParentOb
 };
 
 exports.customMarshalObjectList = [];
-exports.asyncFunctionList = [];
-exports.nativeCallsInterpreterFunctionList = [];
```

Comment:
As far as I could tell my grepping, neither `asyncFunctionList` nor `nativeCallsInterpreterFunctionList` are ever written to. And it appears they were never used when initially added in https://github.com/code-dot-org/code-dot-org/commit/3d9f25c05653e54b53b26f956f42f695bdbd6ff4#diff-11d38f100f6df9c907f72e0cf6bb3671R205